JSON-connect/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
You should be able to remove this file, the parent .gitignore should take care of this.

Reply:
Would you mind if kept it? The files are in a different repo too and I'd prefer not to push the node_modules to that repo.

Reply:
could you do that via a global .gitignore instead? It's not a huge deal, but I'm trying to keep this repo clean and focused if possible. `node_modules` should already be ignored by [this line], right?

[this line]: https://github.com/googledatastudio/community-connectors/blob/master/.gitignore#L44

Reply:
I'm committing to the other repo from within the JSON-connect folder, so the community-connectors/.gitignore is... ignored. Didn't think of the global .gitignore file, thanks for the tip!